libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.WebApi/BotFrameworkConfigurationBuilder.cs

```diff
@@ -27,8 +26,19 @@ public BotFrameworkConfigurationBuilder(BotFrameworkOptions botFrameworkOptions)
         /// <seealso cref="ICredentialProvider" />
         public BotFrameworkConfigurationBuilder UseCredentialProvider(ICredentialProvider credentialProvider)
         {
-            _options.CredentialProvider = credentialProvider;
+            BotFrameworkOptions.CredentialProvider = credentialProvider;
```

Comment:
What value was there in removing the field and changing everything to go through the property accessor? This results in more IL instructions being generated at each call site and more work for the JIT to do. Granted this stuff is in no way on a hot path, but at bare minimum you're still inflating assembly size and startup times with this pattern. 

More importantly perhaps, the work was done already with a field so this caused code churn that needed human eyes put on it for... what?